### ISSUE TITLE
Forcing ipv4 connection over Tunnelblick even though I have both ipv4 and ipv6

### ISSUE THREAD

**Issue by Aracki** [Tunnelblick/Tunnelblick#620]
Is there a way of doing that?

_I've tried with "Disalbe IPv6 unless the VPN server is accessed using IPv6._

Also in both client & server configuration files I have:
```
# Are we connecting to a TCP or UDP server?  Use the same setting as on the
# server.
proto udp6
```

`udp6` should mean the both ipv4 and ipv6 are free to use.

**Comment by jkbullard** [comment]
Please post the diagnostic info obtained by following the instructions at [Read Before You Post](https://tunnelblick.net/cBeforeYouPost.html).

**Comment by Aracki** [comment]
Fixed by upgrading from 2.3 to 2.4
